src/main/java/uk/gov/hmcts/reform/civil/model/docmosis/claimantresponse/JudgmentByAdmissionOrDeterminationMapper.java

```diff
@@ -161,4 +174,166 @@ private String getFormName(CaseData caseData) {
             ? "OCON225"
             : "OCON225a";
     }
+
```

Comment:
Most of the methods added in this mapper seems to be same as in DefaultJudgmentFormGenerator.java file. Should we be moving the common methods to any common util and reuse them in both DJ flow and Judgment By Admission flow?